### ISSUE TITLE
Autoupdating T@H

### ISSUE THREAD

**Issue by tomhughes** [tomhughes/trac-tickets#1100]
**[Submitted to the original trac issue database at 10.43pm, Sunday, 10th August 2008]**

When tah check new version in loop mode then want to update to newest. But This not works very well.
Box is Linux Debian Testing

there is error message

```
This is version 9630 (Rapperswil) of tilesgen running on linux, ID: 1720597620
Died at tilesGen.pl line 126.

```

**Comment by tomhughes** [comment]
**Author: osm@deelkar.net**
**[Added to the original trac issue at 3.17pm, Monday, 11th August 2008]**

can you try r9635?

**Comment by tomhughes** [comment]
**Author: fatbozz@atlas.cz**
**[Added to the original trac issue at 6.25pm, Monday, 11th August 2008]**

Replying to [comment:1 osm@deelkar.net]:

See  the last line. Another error on line 127
```
At revision 9679.
[#0   0% ] tilesGen.pl has changed, re-start new version
- Using working directory /tmp/
- Using process log file tah-process.log
- Uploading with username "fatbozz@atlas.cz"
- Deleting ZIP files after upload
- Configured Layer: captionless
- Configured Layer: tile
- Configured Layer: maplint
- zip is present
- Inkscape version 0.46
- rendering using or/p
- going to use pngcrush
- Pngcrush version 1.6.6
sh: optipng: command not found
- pngnq version 0.5
- Using http://tah.openstreetmap.org/Request/take/ for Requests
This is version 9676 (Rapperswil) of tilesgen running on linux, ID: 1720597620
progstart= does not match option=value at tilesGen.pl line 127.
```

**Comment by tomhughes** [comment]
**Author: deelkar**
**[Added to the original trac issue at 2.12am, Tuesday, 12th August 2008]**

(In [9696]) make sure all needed variables are initialised before re-exec. fixes #1100
